src/tests/Common/CoreCLRTestLibrary/CoreClrConfigurationDetection.cs

```diff
@@ -30,13 +36,11 @@ public static class CoreClrConfigurationDetection
     public static bool IsStressTest =>
         IsGCStress ||
         IsDisableR2R ||
-        IsTailCallStress ||
-        IsJitStressRegs ||
-        IsJitStress ||
-        IsJitMinOpts;
+        IsAnyJitStress ||
+        IsHeapVerify;
 
-    private static string GetEnvironmentVariableValue(string name) =>
-        Environment.GetEnvironmentVariable("DOTNET_" + name) ?? Environment.GetEnvironmentVariable("COMPlus_" + name) ?? "0";
+    private static string GetEnvironmentVariableValue(string name, string defaultValue = "0") =>
```

Comment:
I think adding a default value instead of just hard-coding it is a good practice. I'm just curious as to what prompted this change. Was it simple code cleanliness, or is there something more to it? Just curious.

Reply:
I moved the default into the parameter because we now have a mode whose default is non-zero (TieredCompilation).

Reply:
Okok I understand now. Thanks for clarifying